game/game/gamescript.cpp

```diff
@@ -2643,6 +2671,11 @@ bool GameScript::npc_canseesource(std::shared_ptr<zenkit::INpc> npcRef) {
   return self->canSeeSource();
   }
 
+bool GameScript::npc_isincutscene(std::shared_ptr<zenkit::INpc> npcRef) {
+  auto w = Gothic::inst().world();
+  return w->isCutsceneLock();
```

Comment:
`w` might be `nullptr`, if script calling this function from main-menu

Reply:
I've added a nullptr check